src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -895,5 +895,268 @@ public void SarifTransformerTests_ToVersionOne_NotificationExceptionWithStack()
+                        ""binary"": ""Ig==""
+                      }
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          ""ruleId"": ""WEB1066"",
+          ""message"": {
+            ""arguments"": [
+              ""DIV""
+            ]
+          },
+          ""ruleMessageId"": ""default"",
+          ""locations"": [
+            {
+              ""physicalLocation"": {
+                ""fileLocation"": {
+                  ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                },
+                ""region"": {
+                  ""startLine"": 24,
+                  ""startColumn"": 4,
+                  ""endColumn"": 38,
+                  ""offset"": 803,
+                  ""length"": 34,
+                  ""snippet"": {
+                    ""text"": ""<DIV id=\""test1\"" xweb:fixindex=\""0\""></DIV>""
+                  }
+                }
+              }
+            }
+          ],
+          ""fixes"": [
+            {
+              ""description"": {
+                ""text"": ""Convert tag name to lowercase.""
+              },
+              ""fileChanges"": [
+                {
+                  ""fileLocation"": {
+                    ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                  },
+                  ""replacements"": [
+                    {
+                      ""deletedRegion"": {
+                        ""offset"": 804,
+                        ""length"": 3
+                      },
+                      ""insertedContent"": {
+                        ""binary"": ""ZGl2""
+                      }
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ],
+      ""resources"": {
+        ""rules"": {
+          ""WEB1079.AttributeValueIsNotQuoted"": {
+            ""id"": ""WEB1079"",
+            ""shortDescription"": {
+              ""text"": ""The attribute value is not quoted.""
+            },
+            ""messageStrings"": {
+              ""default"": ""The  value of the '{0}' attribute is not quoted. Wrap the attribute value in single or double quotes.""
+            }
+          },
+          ""WEB1066.TagNameIsNotLowercase"": {
+            ""id"": ""WEB1066"",
+            ""shortDescription"": {
+              ""text"": ""The tag name is not lowercase.""
+            },
+            ""messageStrings"": {
+              ""default"": ""Convert the name of the <{0}> tag to lowercase.""
+            }
+          }
+        }
+      }
+    }
+  ]
+}";
+
+            const string V1LogExpectedText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-1.0.0"",
+  ""version"": ""1.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner""
+      },
+      ""results"": [
+        {
+          ""ruleId"": ""WEB1079"",
+          ""fixes"": [
```

Comment:
The v1 equivalent of v2's `result.message` + `result.ruleMessageId` would be `result.formattedRuleMessage`, which I don't see here.

Reply:
Fixed